src/runtime/virtcontainers/qemu_s390x.go

```diff
@@ -349,8 +349,9 @@ func (q *qemuS390x) appendProtectionDevice(devices []govmmQemu.Device, firmware,
 	case seProtection:
 		return append(devices,
 			govmmQemu.Object{
-				Type: govmmQemu.SecExecGuest,
-				ID:   secExecID,
+				Type:           govmmQemu.SecExecGuest,
+				ID:             secExecID,
+				InitdataDigest: initdataDigest,
```

Comment:
Do we need this even if we don't have any QEMU command line manipulation like TDX and SNP:

https://github.com/kata-containers/kata-containers/blob/9379a18c8ac033b094a5999cb7a30eb3c98aa943/src/runtime/pkg/govmm/qemu/qemu.go#L418-L424

https://github.com/kata-containers/kata-containers/blob/9379a18c8ac033b094a5999cb7a30eb3c98aa943/src/runtime/pkg/govmm/qemu/qemu.go#L509-L515